src/marqo/core/search/hybrid_search.py

```diff
@@ -61,6 +62,7 @@ def search(
                 text_query_prefix: prefix for text queries (for vectorisation only)
                 hybrid_parameters: HybridParameters object to specify all parameters for hybrid search. If not provided,
                     default values will be used.
+                target_hits: target number of hits to return. If None, the default value will be used.
```

Comment:
nit: Is the default value variable or static? If static, can we mention that here?